source/socketstuff.lua

```diff
@@ -149,7 +150,7 @@ function socketstuff.sendToHost()
 	while #clientOutgoingQueue > 0 do
 		if clientOutgoingQueue[1] ~= nil then
 			local serialData = bitser.dumps(clientOutgoingQueue[1])
-			udpClient:send(serialData)
+			local myerr, mymsg = udpClient:send(serialData)

```

Comment:
*cough* :stuck_out_tongue_winking_eye: Maybe just `err` and `msg`? full on `error` and `message` shouldn't be needed, even more so since error is a reserved keyword.